pycbc/events/significance.py

```diff
@@ -192,6 +196,41 @@ def get_n_louder(back_stat, fore_stat, dec_facs,
         **kwargs)
 
 
+def get_far(back_stat, fore_stat, dec_facs,
+            background_time,
+            method=_default_opt_dict['method'],
+            return_counts=False,
+            **kwargs):  # pylint:disable=unused-argument
+    """
+    Return the appropriate FAR given the significance caluclation method
+    """
+    # Get n_louder in background and foreground according to the chosen
+    # method
+    back_cnum, fnlouder = get_n_louder(
```

Comment:
I don't remember why these variables have such ill-matched names.  I guess 'fnlouder' means 'foreground n louder', not sure about 'back_cnum', though. 
Suggest to replace by 'bg_n_louder' and 'fg_n_louder' ?